packages/web-fragments/package.json

```diff
@@ -4,10 +4,13 @@
 	"type": "module",
 	"exports": {
 		"./gateway": {
-			"import": "./src/gateway/index.ts"
+			"import": "./src/gateway/fragment-gateway.ts"
```

Comment:
Please, update the entrypoints as discussed

Reply:
done